app/models/articles.model.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../../db/connection')
+
+exports.fetchArticleById = (article_id)=>{
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({rows})=>{
+        if (rows.length === 0) return Promise.reject({status: 404, msg: 'article not found'})
+        return rows[0]
+    })
+}
+
+exports.fetchArticles = ()=>{
+    const articlePromise = db.query(`SELECT article_id, title, topic, author, created_at, votes FROM articles ORDER BY created_at;`)
+    const commentPromise = db.query(`SELECT article_id FROM comments`)
+    return Promise.all([articlePromise, commentPromise])
```

Comment:
You can do this all in one initial SQL query, which would alleviate the need to make two requests to the database. I would make a note to refactor this at a later point as I think it would be great sql practice!